platform-sdk/swirlds-merkledb/src/main/java/com/swirlds/merkledb/MerkleDbDataSource.java

```diff
@@ -382,7 +387,11 @@ public MerkleDbDataSource(
                 tableConfig.getHashesRamToDiskThreshold());
 
         compactionCoordinator = new MerkleDbCompactionCoordinator(
-                tableName, keyToPathFileCompactor, hashStoreDiskFileCompactor, pathToKeyValueFileCompactor);
+                tableName,
+                keyToPathFileCompactor,
+                hashStoreDiskFileCompactor,
+                pathToKeyValueFileCompactor,
+                database.getConfig());
```

Comment:
If `MerkleDb#getConfig` returned `Configuration` instead of `MerkleDbConfig`, then passing the Configuration object into this constructor wouldn't be needed.

Reply:
So why not change `MerkleDb.getConfig()` to return `Configuration`?